generators/client/templates/angular/src/main/webapp/app/blocks/interceptor/_notification.interceptor.ts

```diff
@@ -18,7 +18,8 @@ export class NotificationInterceptor extends HttpInterceptable {
             let headers = [];
             let i;
             for(i = 0; i < arr.length; i++){
-                if(arr[i][0].endsWith('app-alert') || arr[i][0].endsWith('app-params')) {
+                if(arr[i][0].indexOf('app-alert', arr[i][0].length - 'app-alert'.length) !== -1 ||
```

Comment:
ok, i will remove that change. How about comment below where there are other places with endsWidth - should i scan angularjs *.js for it and fix?